packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -137,9 +136,6 @@ open class DevMenuManager: NSObject, DevMenuManagerProtocol {
   @objc
   @discardableResult
   public func closeMenu() -> Bool {
-    guard let appInstance = appInstance else {
-      return false
-    }
     appInstance.sendCloseEvent()
     return true
   }
```

Comment:
Probably this method no longer needs to return `Bool` value (and be discardable) as it will always return `true` 😉